### PR TITLE
Bl 954 be remove auth from the app

### PR DESCRIPTION
**Pull request by jzkime**
## Description

Removed Okta from most files, such as: package.json, data migrations and seeds, middleware authentication, profileRouter (including profileMiddleware and profileModel), parentRouter and parentModel, and childRouter. Also checked that responses were relevant to README.md file.

Fixes # BE: Remove auth from the app

## Loom Video
https://www.loom.com/share/7d5ae93de6654ab3bcfc821f6ebba040

## Type of change

Please delete options that are not relevant.

- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by jzkime** [comment]
@paulstgermain, Ryan explained to me he wanted to give credit for what was done, since the Backend is going to be Java.

File: api/children/childrenRouter.js [hunk 1]
```diff
@@ -30,6 +30,8 @@ router.post(
   checkChildObject,
   async function (req, res, next) {
     const { profile_id } = req.profile;
+    if (!profile_id)
```
Comment (Jwatk13) [inline api/children/childrenRouter.js:33]: Nice fill in function!

**Review by Jwatk13 — APPROVED** [review]
Looks good overall, I understand the changes and what the changes do. Great job!

**Comment by gclancy121** [comment]
good job clearing out okta

**Comment by ryan-hamblin** [comment]
> @paulstgermain, Ryan explained to me he wanted to give credit for what was done, since the Backend is going to be Java.

Hey @paulstgermain can we assign the expected Story Points for @jzkime here? They are still waiting on the points to be accredited to their Labs Journey.

**Comment by JessWillCode** [comment]
Hey, @ashtilawat23 I just checked and the only things failing the sonar test are variables that are not used. Could we go ahead and merge this so I can pull it down?

**Pull request merged**